locales/en/messages.po

```diff
@@ -2087,39 +2100,43 @@ msgstr "Your NoRent letter and important next steps"
 msgid "Your NoRent.org letter"
 msgstr "Your NoRent.org letter"
 
-#: frontend/lib/rh/rental-history.tsx:191
+#: frontend/lib/rh/rental-history.tsx:235
 msgid "Your Rent History has been requested from the New York State DHCR!"
 msgstr "Your Rent History has been requested from the New York State DHCR!"
 
 #: frontend/lib/norent/letter-builder/post-signup-no-protections.tsx:7
 msgid "Your account is set up"
 msgstr "Your account is set up"
 
-#: frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx:316
+#: frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx:315
 msgid "Your apartment may be rent stabilized."
 msgstr "Your apartment may be rent stabilized."
 
-#: frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx:317
+#: frontend/lib/rh/rental-history.tsx:138
+msgid "Your building had in {0}, according to property tax documents."
+msgstr "Your building had in {0}, according to property tax documents."
+
```

Comment:
@toolness see here! This should read "Your building had {0, plural, one...." but instead the whole plural tag is left out!

Reply:
holy cow this is truly messed up

Reply:
I knoww! Let's wait until next week to figure it out?

Reply:
um i found out the problem... or rather i found a solution, i have no idea what the underlying problem is.  Apparently  it has to do with the ORDER in which we import things from `@lingui/macro`... if we do this:

```js
import { Plural, Trans } from "@lingui/macro";
```

the `<Plural>` tag does not work. But if we change it to this:

```js
import { Trans, Plural } from "@lingui/macro";
```

everything works fine?!!!1111

Reply:
Oy ok this is logged in https://github.com/lingui/js-lingui/issues/372#issuecomment-637783760 and is apparently a known issue with Lingui.  It's fixed in Lingui v3, which ... whoa appears to have finally been released!!!  I've filed #1782 so we upgrade to it and won't be bitten by this again in the future.

Reply:
oh wow amazing, you are really saving the day today!